packages/react-composites/src/composites/CallComposite/components/LocalAndRemotePIP.tsx

```diff
@@ -93,14 +101,39 @@ export const LocalAndRemotePIP = (props: LocalAndRemotePIPProps): JSX.Element =>
     [ariaLabel]
   );
 
+  // If there are no remote participants, show the local participant as the primary tile
+  const primaryTileProps: _PictureInPictureInPictureTileProps = useMemo(
+    () => ({
+      content: remoteVideoTileProps ? (
+        <_RemoteVideoTile {...remoteVideoTileProps} />
+      ) : (
+        <_LocalVideoTile {...localVideoTileProps} />
+      ),
+      // TODO: when the calling SDK provides height/width stream information - update this to reflect the stream orientation.
+      orientation: 'portrait'
+    }),
+    [localVideoTileProps, remoteVideoTileProps]
+  );
+
+  // If we are showing the local participant as the primary tile, show nothing for the secondary tile
+  const secondaryTileProps: _PictureInPictureInPictureTileProps | undefined = useMemo(
+    () =>
+      remoteVideoTileProps
+        ? {
+            content: <_LocalVideoTile {...localVideoTileProps} personaMinSize={20} />,
+            // TODO: when the calling SDK provides height/width stream information - update this to reflect the stream orientation.
+            orientation: 'portrait'
+          }
+        : undefined,
+    [localVideoTileProps, remoteVideoTileProps]
+  );
+
   return (
     <_PictureInPictureInPicture
       onClick={props.onClick}
       strings={strings}
-      // If there are no remote participants, show the local participant as the primary tile
-      primaryTile={remoteVideoTile ?? localVideoTile}
-      // If we are showing the local participant as the primary tile, show nothing for the secondary tile
-      secondaryTile={remoteVideoTile ? localVideoTile : undefined}
+      primaryTile={primaryTileProps}
```

Comment:
Consider an alternative API:

```
<_PictureInPictureInPicture
  onClick={...}
  strings={...}
  onRenderPrimaryTile={remoteVideoTileProps ? <_RemoteVideoTile {...remoteVideoTileProps} /> : <_LocalVideoTile {...localVideoTileProps} />}
  primaryTileOrientation='portrait'
  onRenderSecondaryTileProps={remoteVideoTileProps ? <_LocalVideoTile {...localVideoTileProps} /> : undefined}
  secondaryTileOrientation='portrait'
>
```

I _think_ this avoids the rendering issues we observed yesterday because the `onRender*` functions directly use a component.

Benefits of this API: No need to memoize `localVideoTileProps` and `remoteVideoTileProps` because they are all destructured

Drawbacks of this API: Namespacing of other props like `primaryTileOrientation` etc. This is a common React problem. You need to flatten objects out to avoid the need to memoize.

Just a soft suggestion.

Reply:
I had this at one point but reverted because `secondaryTileOrientation` becomes moot if `onRenderSecondaryTileProps` is not supplied. Which I realize we do a lot in other places but it feels poor. If we add more tile props then they continue needing to be flat and we have a growing number of potentially redundant props
The way it is currently avoids the rendering issue - but does move the `<RemoteVideoTile>` outside of the return which isn't ideal - gonna leave for now but its all internal so can be easily updated in future